src/store/getters/state-getters.ts

```diff
@@ -551,3 +551,14 @@ export const showFeeSummary = (state: StateIF): boolean => {
   )
   return (haveFilingChange && !isEqual(getFilingData(state), defaultFilingData))
 }
+
+/** Return a boolean indicating if the minimum share classes requirements are met. */
+export const invalidMinimumShareClass = (state: StateIF): boolean => {
+  const shareClasses = state.stateModel.shareStructureStep.shareClasses
+
+  // Filter out REMOVED class actions
+  const currentShareClasses = shareClasses.filter(x => x.action !== ActionTypes.REMOVED)
+    .map((x) => { const { action, ...rest } = x; return rest })
```

Comment:
This is correct but maybe unnecessary... can you remove the `map`, since you only care about the number of them, not actually what they are?
(less work done = faster code)

Reply:
I think you are right, most likely could! Will verify 👍

Reply:
Indeed, works good.
Good change and good catch, thanks!

Reply:
Updated